pom.xml

```diff
@@ -106,12 +107,17 @@
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>azure-spring-cloud-feature-management</artifactId>
+        <version>${version.azure-spring-cloud-feature-management}</version>
+      </dependency>
```

Comment:
How about deleting this? Because `spring-cloud-azure-feature-management-web` will bring it in.